build.gradle

```diff
@@ -729,6 +730,7 @@ project(':iceberg-parquet') {
     api project(':iceberg-api')
     implementation project(':iceberg-core')
     implementation project(':iceberg-common')
+    implementation 'com.github.rdblue:brotli-codec:0.1.1'
```

Comment:
could we do compile-only for this?

Reply:
we require this dependency in runtime, hence added it as implementation, we load the the Codec class dynamically via - 
```
      Class<?> codecClass;
      try {
        codecClass = Class.forName(codecClassName);
      } catch (ClassNotFoundException e) {
        // Try to load the class using the job classloader
        codecClass = configuration.getClassLoader().loadClass(codecClassName);
      }
```

Hence just having compileOnly dependency might not work, are you recommending we should not include jar and update the iceberg doc instead ?